src/index.ts

```diff
@@ -215,7 +215,7 @@ function createTerminal() {
   }
 
   function stringify(obj: any) {
-    return typeof obj === 'object' ? `${JSON.stringify(obj)}` : obj.toString()
+    return ['object', 'undefined'].includes(typeof obj) ? `${JSON.stringify(obj)}` : obj.toString()
```

Comment:
I think we should just do:
```suggestion
    return JSON.stringify(obj)
```
Just checked and for example in the case of `new Date()`, `console.log` seems to be using `JSON.stringify` instead of `.toString()`.